drivers/serial/uart_native_tty.c

```diff
@@ -220,8 +243,11 @@ static struct uart_driver_api native_tty_uart_driver_api = {
 			},                                                                         \
 	};                                                                                         \
                                                                                                    \
+	static struct uart_config uart_config_##inst = native_tty_##inst##_cfg.uart_config;             \
```

Comment:
default to values of the read-only init config, in case there was no `configure` call, but just `init`